packages/mds-metrics-service/orm.ts

```diff
@@ -0,0 +1,76 @@
+/*
+    Copyright 2019-2020 City of Los Angeles.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+ */
+
+import { ConnectionManager } from '@mds-core/mds-orm'
+import { InsertReturning } from '@mds-core/mds-orm/types'
+import { DeepPartial, Between, In } from 'typeorm'
+import { timeframe } from '@mds-core/mds-utils'
+import { SingleOrArray } from '@mds-core/mds-types'
+import ormconfig from './ormconfig'
+import { MetricEntity } from './entities'
+import { ReadMetricsRequiredParameters, ReadMetricsOptionalParameters } from './types'
+
+const manager = ConnectionManager(ormconfig)
+
+export const initialize = async () => manager.initialize()
+
+const singleOrArrayFilter = <T>(filter: string, value: SingleOrArray<T> | undefined) => {
```

Comment:
I could definitely see this being useful elsewhere, think we should move it to mds-utils at some point.

Reply:
Not `mds-utils`, but maybe `mds-orm/utils`